src/sqlfluff/dialects/dialect_exasol.py

```diff
@@ -985,9 +985,10 @@ class CreateTableStatementSegment(BaseSegment):
                         Ref("TableOutOfLineConstraintSegment"),
                         Ref("CreateTableLikeClauseSegment"),
                         Ref("CommaSegment", optional=True),
-                        Ref("TableDistributionPartitonClause", optional=True),
                         min_times=1,
                     ),
+                    Ref("CommaSegment", optional=True),
```

Comment:
Does this make sense without the following `TableDistributionPartitonClause`?

I.e. should they both be in a sequence together that is optional, rather than both be optional individually?

Reply:
The `CommaSegment` is not needed at all. It comes from the `AnyNumberOf` sequence above.

Reply:
Hmmm.... the only issue is this allows the following:

Trailing comma:
```sql
CREATE TABLE t5 (
    a DOUBLE,
    b DOUBLE,
);
```

and No commas:

```sql
CREATE TABLE t5 (
    a DOUBLE
    b DOUBLE
);
```

and no comma before the `TableDistributionPartitonClause`:

```sql
CREATE TABLE t5 (
    a DOUBLE,
    b DOUBLE
    DISTRIBUTE BY a
);
```

All of which are invalid Exasol SQL I presume?

However, I've played around with this and can't get it to easily prevent this (mostly due to #1366).

So maybe best merging this as it is?

Reply:
You're correct, i noticed the same. I couldn't find out how to prevent this, too (at least the DB engine will stop this, as its no vaild statement). But I think this bug happens also at the current version. IMHO we should merge this and keep track on this in a new issue.